helpers/sortByMapping.js

```diff
@@ -7,34 +7,51 @@ module.exports = function( dust ) {
 
   /*
   * @description Sorting method that orders a list of things based on a provided map
-  * @param {object} eventsObject Object to sort
-  * @param {string} map Map to use to sort with
-  * @example {@_sortByMapping eventsObject=eventsReply.events map="ASHLKE, ASHWKE, ASHJBE} {/_sortByMapping} output loop of events
+  * @param {object} sortObject - Object to sort
+  * @param {string} map - Map to use to sort with
+  * @param {string} indexes - index structure of response to sort on
```

Comment:
I have updated this, take a look and if it still doesn't make sense perhaps we can sit and come up with something.